scripts/run_dev.sh

```diff
@@ -80,7 +80,7 @@ fi
 # Check if all LFS files are in place
 git rev-parse &>/dev/null
 if [[ $? -eq 0 ]]; then
-    LFS_FILES_STATUS=$(cd $ISAAC_ROS_DEV_DIR && git lfs ls-files | cut -d ' ' -f2)
+    LFS_FILES_STATUS=$(cd "$ISAAC_ROS_DEV_DIR/src/isaac_ros_common" && git lfs ls-files | cut -d ' ' -f2)
```

Comment:
There's no guarantee here about the the relative location of isaac_ros_common and ISAAC_ROS_DEV_DIR, though. "src" is not required technically.

Reply:
This can be move to $ROOT as we suspect that the run_dev.sh script itself is in isaac_ros_common git repo.

Reply:
This patch assumes that users will follow the official tutorial, which allows us to assume the relative path of `isaac_ros-dev` and `isaac_ros_common` is `isaac_ros-dev/src/isaac_ros_common`.

The support of arbitrary directory hierarchy (e.g., excluding `src`) is not considered in this patch. A fix that allows arbitrary directory hierarchy will be more complicated, which deserves another patch.

<hr>

I don't think we can move this modification to `$ROOT`, since `$ROOT` itself is used multiple times throughout the script. For example:

https://github.com/NVIDIA-ISAAC-ROS/isaac_ros_common/blob/88023df7376ac02421302a61349e439165557778/scripts/run_dev.sh#L12

requires `$ROOT` to be set to `$ISAAC_ROS_DEV_DIR/src/isaac_ros_common/scripts`.

On the other hand, `$ISAAC_ROS_DEV_DIR` is required to be set to `<SOME_PATH>/isaac_ros-dev` since it will be used when launching a container:

https://github.com/NVIDIA-ISAAC-ROS/isaac_ros_common/blob/88023df7376ac02421302a61349e439165557778/scripts/run_dev.sh#L183